libafl/src/lib.rs

```diff
@@ -6,7 +6,7 @@ Welcome to `LibAFL`
 #![cfg_attr(feature = "document-features", doc = document_features::document_features!())]
 #![no_std]
 // For `type_eq`
-#![cfg_attr(nightly, feature(specialization))]
+// #![cfg_attr(nightly, feature(specialization))]
```

Comment:
I think we should keep this guy around since the specialization version is the more sound one (i.e., it doesn't abuse UB in the std library)

Reply:
Nevermidn we don't anymore :D